packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -319,23 +373,54 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget>
     _onInspectorSelectionChanged();
   }
 
-  void _updateProperties() async {
-    if (_previousProperties != null) {
-      entranceController.reverse();
-    } else {
-      entranceController.forward();
-    }
-  }
-
-  Future<void> _onTap(LayoutProperties properties) async {
-    highlighted = properties;
-    final node = properties.node;
+  Future<void> _onTap(LayoutProperties selected) async {
+    highlighted = selected;
```

Comment:
TODOs for me is to explain why I'm not calling the setState here
It is because we are calling refreshSelection which will trigger selection event that causes this widget to be notified and do setState from that

Reply:
nevermind, this `highlighted = properties` statement is actually redundant.